bin/oref0-pushover.sh

```diff
@@ -80,14 +80,14 @@ fi
 
 #function pushover_snooze {
 # check Nightscout to see if another rig has already sent a carbsReq pushover recently
-    URL=$NIGHTSCOUT_HOST/api/v1/devicestatus.json?count=100
+    URL=$NIGHTSCOUT_HOST/api/v1/devicestatus.json?count=20
```

Comment:
Do you use pushover? If not, should we disable this if pushover isn't configured?

Reply:
I'm reluctant to reduce this value, as for our use case, with multiple rigs all uploading devicestatus data, we might get more than 20 devicestatus entries between pushovers. I'd have to check. If we can just disable it for people who don't use it, that'd be easier.